### PR TITLE
added gtk4, helvum

### PR DESCRIPTION
**Pull request by kilea-chan**
2 new packages for the repo

### CODE REVIEW

File: gnome/gtk4/build.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+pkgname=gtk
```
Comment (Ella-0) [inline gnome/gtk4/build.sh:2]: I think we should add a maintainer line here such as `#m star`

File: snd/helvum/build.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+pkgname=helvum
```
Comment (Ella-0) [inline snd/helvum/build.sh:2]: here too :3 maybe not idk what do you think?

**Review by Ella-0 — CHANGES_REQUESTED** [review]
Added a couple of comments about adding maintainer lines. other than that LGTM

File: snd/helvum/build.sh (hunk 2 above)
Comment (kilea-chan) [reply snd/helvum/build.sh:2]: Will do

**Review by Ella-0 — APPROVED** [review]

**Pull request merged**